src/vm/i386/excepx86.cpp

```diff
@@ -38,6 +38,7 @@
 #include "asmconstants.h"
 #include "virtualcallstub.h"
 
+#ifndef WIN64EXCEPTIONS
```

Comment:
There are too many functions just to try to go through them one by one. But looking at the excepamd64.cpp and exceparm.cpp, it looks like only a few functions are needed here. So I would recommend wrapping the whole body of the source file (starting after the includes) in #ifndef WIN64EXCEPTIONS and then moving out if that only functions that you'll find as missing. I guess it should be about 5 functions, CLRNoCatchHandler being one of them.